example.env

```diff
@@ -0,0 +1,18 @@
+BS_LANG=en
+BS_URL=http://localhost
+BS_USER=WikiSysop
+BS_PASSWORD=PleaseChangeMe
+HTTP_PORT=80
+HTTPS_PORT=443
+IMAGE_NAME=bluespice/bluespice-free
+IMAGE_TAG=4.2.x
+DISABLE_PINGBACK=yes
```

Comment:
Is this still used? If so, maybe another default value could be used.

Reply:
Do you mean for the disable pingback var?

Reply:
Or all the vars

Reply:
the ping was used in previous version, so I have updated it to the same (it will send ping by default now)